src/Aspire.Dashboard/Components/Controls/ExceptionDetails.razor

```diff
@@ -1,52 +1,35 @@
 ﻿@namespace Aspire.Dashboard.Components
 
 @using Aspire.Dashboard.Extensions
+@using Aspire.Dashboard.Components.Dialogs
 @using Aspire.Dashboard.Otlp.Model
 @using Aspire.Dashboard.Resources
 
 @inject IStringLocalizer<ControlsStrings> Loc
 
-@{
-    var iconId = Guid.NewGuid().ToString();
-    @*
-     * We don't want a browser tooltip to display when hovering over the error icon.
-     * Use alt instead of title for screen readers. Also, the column has a title set
-     * so add a whitespace title to prevent a browser tooltip from displaying.
-     *@
-    var title = " ";
-}
-
 @if (!string.IsNullOrWhiteSpace(ExceptionText))
 {
-    <FluentIcon alt="@Loc[nameof(ControlsStrings.ExceptionDetailsTitle)]"
-                title="@title"
-                Style="flex-shrink: 0"
-                Id="@iconId"
-                Icon="Icons.Filled.Size16.DocumentError"
-                Color="Color.Accent"
-                Class="severity-icon" />
-
-    <FluentTooltip Anchor="@iconId" MaxWidth="650px">
-        <div style="padding: 10px">
-            <h4>@Loc[nameof(ControlsStrings.ExceptionDetailsTitle)]</h4>
-
-            <pre style="white-space: pre-wrap; overflow-wrap: break-word; max-height: 300px; overflow: auto;">@ExceptionText</pre>
-
-            <div style="float: right; margin-bottom: 10px;">
-                <FluentButton Id="@Guid.NewGuid().ToString()"
-                              Appearance="Appearance.Lightweight"
-                              AdditionalAttributes="@FluentUIExtensions.GetClipboardCopyAdditionalAttributes(ExceptionText, Loc[nameof(ControlsStrings.GridValueCopyToClipboard)].ToString(), Loc[nameof(ControlsStrings.GridValueCopied)].ToString())">
-                    @Loc[nameof(ControlsStrings.GridValueCopyToClipboard)]
-
-                    <FluentIcon Class="copy-icon align-text-bottom" Style="display:inline;" Icon="Icons.Regular.Size16.Copy" />
-                    <FluentIcon Class="checkmark-icon align-text-bottom" Style="display:none;" Icon="Icons.Regular.Size16.Checkmark" />
-                </FluentButton>
-            </div>
-        </div>
-    </FluentTooltip>
+    <FluentButton Appearance="Appearance.Lightweight"
```

Comment:
It doesn't look good on mouse over with the button hard up against the text:

![image](https://github.com/user-attachments/assets/179011ab-1d8e-4561-87fb-0b351d7b7b10)

Reduce the width of the button (maybe have 4px?) and add a right margin to provide some space between the button and text.

Example from the inline error button on resources page:

https://github.com/dotnet/aspire/blob/fe53c9874e5db7e64be5221b5f41c0ea1d9203b0/src/Aspire.Dashboard/Components/Pages/Resources.razor.css#L6-L11

Reply:
I called this out in:

> Here's how it looks on mouse-over, along with a tooltip. The button appears too close to the text here, but that's consistent with other inline lightweight buttons. It only shows on mouse over. If we add more margin, it looks bad when the mouse isn't hovering over it.

Making this button smaller means it no longer matches the equivalent button on the right. We should be consistent.

Reply:
Do you mean the `...` button that appear on hover? I wouldn't call it inline. It appears over the top of content on hover. The existing prior art of a clickable icon that is inline in text is the unread logs errors button. It doesn't have padding, just a margin to give it space from text (CSS above).

The problem with having the large padding is content isn't consistently aligned with the start of the column. It doesn't look good. And it breaks consistency with other columns that start with an icon (e.g. the level column also in the structured logs table).

Aspire today:

![image](https://github.com/user-attachments/assets/ac3ea25c-3291-4552-a045-3c3940d82494)

Current PR:

![image](https://github.com/user-attachments/assets/193b5721-8fcb-4d6c-8941-c30df7ad68f2)

Appearance wise it should stay the same, with the change in behavior from hover to click.

cc @davidfowl @leslierichardson95 for input.

Reply:
Honestly, I would prefer if we removed that little box and moved it to where the actions are on the left. Similar to how start/stop look on the resources view.

Reply:
Hmm, that's an interesting idea. One problem I see with that is if there is an error then the `...` actions button would have to always be shifted right to make room for the "View exception details" button. But a lot of the time there won't be any logs exception details visible in the UI, so the `...` button will look like it is in the wrong place.

Here is an alternative idea:

What about changing the button in this PR to just a regular icon. No hovering, no clicking. Then in the actions menu, there is a "Exception details" option. There is already an menu option for viewing the message in the text visualizer. Would be simple to do the same thing for the exception details.

Imagine an extra option for "Exception details" with its matching icon in the picture below:
![image](https://github.com/user-attachments/assets/8d596991-e463-4e68-98e1-1e4c6f748483)

Reply:
Prefer if it was aligned. A button without a tooltip feels bad though. It's hard to tell what that icon even is until you hover over it. @DamianEdwards was lamenting that the experience of viewing errors in the structured logs felt difficult compared to looking at console logs.

I think the core fix here is to add the menu item to view the exception details in the grid view *and* in the details view. If we can do the same with the icon then great, but its less discoverable IMO.

Reply:
How about changing the status icon instead of resources with unread errors, rather than trying to fit an extra icon in somewhere else?

Reply:
I moved the button to the right hand side of the "cell" so that it sits alongside the masking button and the ... dropdown menu button. There's a generic facility for us to add other buttons there in future.

Reply:
See updated screenshots in description.